fanuc_ros_inteface:
  ros__parameters:
    buffer_length_param: 8
    control_time: 0.04
    robot_ip: "10.11.31.111"




    